thing-url-adapter.js

```diff
@@ -71,12 +101,10 @@ class ThingURLProperty extends Property {
       return Promise.resolve(value);
     }
 
+    const header = getHeaders(this.url, true);
```

Comment:
nit: all of these calls should be `headers = getHeaders()`, not `header`

Reply:
I should have known better.